config/galaxy.ini.sample

```diff
@@ -634,7 +634,7 @@ nglims_config_file = tool-data/nglims.yaml
 # also causes the files used by PBS/SGE (submission script, output, and error)
 # to remain on disk after the job is complete.  Debug mode is disabled if
 # commented, but is uncommented by default in the sample config.
```

Comment:
yeah I know, I am working on PR, just didn't want to start the debate here

On Mon, Oct 5, 2015 at 10:10 AM Nicola Soranzo <notifications@github.com>
wrote:

> In config/galaxy.ini.sample
> <https://github.com/galaxyproject/galaxy/pull/840#discussion_r41147451>:
>
> > @@ -634,7 +634,7 @@ nglims_config_file = tool-data/nglims.yaml
> >  # also causes the files used by PBS/SGE (submission script, output, and error)
> >  # to remain on disk after the job is complete.  Debug mode is disabled if
> >  # commented, but is uncommented by default in the sample config.
>
> Too fast merge, the comment above (is uncommented by default in the
> sample config.) was not updated accordingly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/galaxyproject/galaxy/pull/840/files#r41147451>.
>